earthfile2llb/converter.go

```diff
@@ -2395,7 +2395,7 @@ func (c *Converter) imageVertexPrefix(id string, platform platutil.Platform) str
 }
 
 func (c *Converter) vertexPrefixWithURL(url string) string {
-	return fmt.Sprintf("[%s(%s) %s] ", c.mts.Final.Target.String(), url, url)
```

Comment:
printing the url twice seemed redundant, I can put this back if there is a good reason I was unaware of